app/models/manageiq/providers/nuage/network_manager/refresher.rb

```diff
@@ -9,5 +9,37 @@ def parse_legacy_inventory(ems)
     def post_process_refresh_classes
       []
     end
+
+    def collect_inventory_for_targets(ems, targets)
+      targets_with_data = targets.collect do |target|
+        target_name = target.try(:name) || target.try(:event_type)
+
+        _log.info("Filtering inventory for #{target.class} [#{target_name}] id: [#{target.id}]...")
```

Comment:
Despite the fact that other providers also doesn't seem to have `log_header` here, I suggest you add it.